src/Uno.Toolkit.UI/Controls/LoadingView/LoadingView.cs

```diff
@@ -64,6 +67,25 @@ public object LoadingContent
 
 		#endregion
 
+		#region DependencyProperty: LoadingContentTemplate
+
+		public static DependencyProperty LoadingContentTemplateProperty { get; } = DependencyProperty.Register(
+			nameof(LoadingContentTemplate),
+			typeof(DataTemplate),
+			typeof(LoadingView),
+			new PropertyMetadata(default(object)));
+
+		/// <summary>
+		/// Gets or sets the content template to be used when displayin LoadingContent during loading/waiting.
+		/// </summary>
+		public object LoadingContentTemplate
```

Comment:
I don't know the right answer to this but I usually see "ContentTemplate" properties being of type `DataTemplate`. Just curious if there is a reason to not use `DataTemplate`

Reply:
Corrected